pkg/apis/core/types_shoot.go

```diff
@@ -497,6 +497,37 @@ type KubeAPIServerConfig struct {
 	EnableAnonymousAuthentication *bool
 	// EventTTL controls the amount of time to retain events.
 	EventTTL *metav1.Duration
+	// AccessControl provides authorization mechanisms for Shoot-API-Server.
+	AccessControl *AccessControl
+}
+
+// AccessControl provides authorization mechanisms for Shoot-API-Server.
+// Note: The schema (incl. child structs) and documentation resembles istio's AuthorizationPolicy.
```

Comment:
Maybe it is better to add a hint that the structure resembles a simplified version of istio's AuthorizationPolicy. When I first looked at the istio AuthorizationPolicy for comparison I was rather confused by the statement.
```suggestion
// Note: The schema (incl. child structs) and documentation resembles istio's AuthorizationPolicy, but is heavily simplified.
```